src/server/worker.ts

```diff
@@ -174,12 +182,26 @@ export const getScheduler = async (attempt = 0): Promise<Scheduler> => {
   ];
 
   if (config.ENABLE_AUTOSENDING) {
-    schedules.push({
-      name: QUEUE_AUTOSEND_INITIALS_TASK_IDENTIFIER,
-      taskIdentifier: QUEUE_AUTOSEND_INITIALS_TASK_IDENTIFIER,
-      pattern: "*/1 * * * *",
-      timeZone: config.TZ
-    });
+    schedules.push(
+      {
+        name: QUEUE_AUTOSEND_INITIALS_TASK_IDENTIFIER,
+        taskIdentifier: QUEUE_AUTOSEND_INITIALS_TASK_IDENTIFIER,
+        pattern: "*/1 * * * *",
+        timeZone: config.TZ
+      },
+      {
+        name: PAUSE_AUTOSENDING_CAMPAIGNS_TASK_IDENTIFIER,
+        taskIdentifier: PAUSE_AUTOSENDING_CAMPAIGNS_TASK_IDENTIFIER,
+        pattern: "10 * * * *",
```

Comment:
Question: https://rewiredcoop.slack.com/archives/C02LYSTHP0Q/p1668811954270619?thread_ts=1667845110.462929&cid=C02LYSTHP0Q

Reply:
Got it! I think 10 mins before the hour (when the [unqueue](https://github.com/politics-rewired/Spoke/pull/1533/files#diff-eae205c288039f53fdaee668619c15cd9193fdcff31a1a632a5db5d59c9a97b8R201) currently happens) so that we're consistent with [when initials are assignable to texters](https://github.com/politics-rewired/Spoke/blob/main/schema-dump.sql#L1689)